runtime/src/next/mod.rs

```diff
@@ -288,8 +255,8 @@ impl Router {
         tokio::task::spawn_blocking(move || {
             let mut iter = logs_stream.bytes().filter_map(Result::ok);
 
-            while let Some(log) = Log::from_bytes(&mut iter) {
-                logs_tx.blocking_send(Ok(log.into())).expect("to send log");
+            while let Some(_log) = Log::from_bytes(&mut iter) {
+                // TODO: send to the log server here
```

Comment:
We can follow up here to send logs in a separate PR for a `shuttle-next` runtime, but we should open a ticket to track this.

Reply:
Indeed, will do!

Reply:
ENG-899 opened for this.